packages/discord.js/src/managers/CategoryChannelChildManager.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const CachedManager = require('./CachedManager');
+const GuildChannel = require('../structures/GuildChannel');
+
+/**
+ * Manages API methods for CategoryChannels' children and stores their cache.
+ * @extends {CachedManager}
+ */
+class CategoryChannelChildManager extends CachedManager {
```

Comment:
Shouldn't this extend `DataManager`? All `CachedManager` brings is a private readonly `_cache` property (being a map) and the private `_add` method, none of which is used here.

Using `DataManager` would allow us to skip allocating a Collection every time an `CategoryChannelChildManager` is created.

Reply:
I just extended CachedManager because this has a cache, although it isn't used directly so I guess I could change that yeah